site/content/docs/cli/installation.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Installation
```

Comment:
Why's there `Installation` and `Getting started`? Couldn't we move all this into getting started and call it? 🫣

Reply:
`Installation` is a longer topic if you want to do it from scratch, that's why I split it.